docs/install.md

```diff
@@ -2,30 +2,16 @@
 
 **NOTE**: Currently the child chain server and watcher are bundled within a single umbrella app.
 
-Only **Linux** platforms are supported now. These instructions have been tested on a fresh Linode 2048 instance with Ubuntu 16.04.
+Only **Linux** and **OSX** platforms are supported now. These instructions have been tested on a fresh Linode 2048 instance with Ubuntu 16.04.
 
 ## Prerequisites
-* **Erlang OTP** `>=20` (check with `elixir --version`)
-* **Elixir** `>=1.6` (check with `elixir --version`)
-* **solc** `~>0.5` (check with `solc --version`)
-
-### Optional prerequisites
-* **`httpie`** - to run HTTP requests from `docs/demoxx.md` demos
+* **Erlang OTP** `>=21` (check with `elixir --version`)
+* **Elixir** `>=1.8.2` (check with `elixir --version`)
```

Comment:
A small reservation against the `>` part because there was an occasion where someone tried `1.9.*` and it didn't work due to some behavioral differences. Maybe `=1.8.*` instead? The install instructions below pin to `elixir=1.8.2-1` anyway.